build/accelerate_launch.py

```diff
@@ -117,18 +128,91 @@ def main():
             sys.exit(INTERNAL_ERROR_EXIT_CODE)
 
         try:
+            last_checkpoint_dir = get_highest_checkpoint(tempdir)
+            last_checkpoint_path = os.path.join(tempdir, last_checkpoint_dir)
+
+            use_flash_attn = job_config.get("use_flash_attn", True)
+            adapter_config_path = f"{last_checkpoint_path}/adapter_config.json"
```

Comment:
It would be better to use `os.path.join` here since we use it in most places and it's easier to port to different platforms :)
```suggestion
            adapter_config_path = os.path.join(last_checkpoint_path, "adapter_config.json")
```